short_answer/static/html/short_answer.html

```diff
@@ -9,7 +9,7 @@
     {% if feedback %}
     <div class="feedback {% if not answer %}hidden{% endif %}"><p>{{ feedback }}</p></div>
     {% endif %}
-    <button type="submit" class="btn-submit" {% if passed_due %}disabled{% endif %}>{% trans "Submit" %}</button>
+    <button id="answer-submission-button" type="submit" class="btn-submit" {% if passed_due %}disabled{% endif %}>{% trans "Submit" %}</button>
```

Comment:
There can be more than one instance of this xblock per page which will result in duplicate id's, which is never a good thing to have.
Also there is an empty id in the textarea element above that you can probably remove.